envergo/moulinette/models.py

```diff
@@ -1837,6 +1837,11 @@ def summary(self):
             haies = self.catalog["haies"]
             summary["longueur_detruite"] = haies.length_to_remove()
             summary["longueur_plantee"] = haies.length_to_plant()
+            hedge_centroid_coords = haies.get_centroid_to_remove()
+            summary["lnglat_centroide_haie_detruite"] = (
```

Comment:
Est-ce qu'il n'y a pas une petite typo ?

```suggestion
            summary["lnglat_centroid_haie_detruite"] = (
```

Reply:
En fait je l'ai appelé partout "lnglat_centroide_haie_detruite", avec centroïde en français : c'est pour matomo donc je sais pas si ça a une grande importance.

Reply:
Oui effectivement, c'est très bien comme ça.